inc/class-core-sitemaps.php

```diff
@@ -118,7 +118,7 @@ public function register_rewrites() {
 		add_rewrite_tag( '%sub_type%', '([^?]+)' );
 
 		// Register index route.
-		add_rewrite_rule( '^sitemap\.xml$', 'index.php?sitemap=index', 'top' );
+		add_rewrite_rule( '^wp-sitemap\.xml$', 'index.php?sitemap=index', 'top' );
```

Comment:
`index.php?sitemap=index` could also collide with other plugins, why not also use the `wp-` prefix: `index.php?wp-sitemap=index` ?

Reply:
Mind creating an issue for that?

Reply:
sure!